extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -226,6 +244,29 @@ export class AmpPanZoom extends AMP.BaseElement {
     return ELIGIBLE_TAGS[element.tagName];
   }
 
+  /**
+   * Creates zoom buttoms
+   */
+  createZoomButton_() {
+    this.zoomButton_ = this.element.ownerDocument.createElement('div');
+    this.zoomButton_.classList.add('i-amphtml-pan-zoom-in-button');
+    this.zoomButton_.classList.add('i-amphtml-pan-zoom-button');
+    this.zoomButton_.addEventListener('click', () => {
+      if (this.zoomButton_.classList.contains('i-amphtml-pan-zoom-in-button')) {
+        this.transform(0, 0, this.maxScale_);
+        this.zoomButton_.classList.remove('i-amphtml-pan-zoom-in-button');
+        this.zoomButton_.classList.add('i-amphtml-pan-zoom-out-button');
+      } else {
+        this.transform(0, 0, this.minScale_);
+        this.zoomButton_.classList.add('i-amphtml-pan-zoom-in-button');
+        this.zoomButton_.classList.remove('i-amphtml-pan-zoom-out-button');
+      }
+    });
+    this.mutateElement(() => {
```

Comment:
I don't think this needs to be in mutateElement since it is called sync from layoutCallback. (the transform needs to be though, I assume `set_` uses mutate?